app/src/main/res/values/strings.xml

```diff
@@ -718,4 +718,56 @@
     <string name="sign_out_cancel">Cancel</string>
     <!-- Error message snackbar shown after the user tried to select a default folder which cannot be altered -->
     <string name="bookmark_cannot_edit_root">Can’t edit default folders</string>
+
+    <!-- Enhanced Tracking Protection -->
+    <!-- Link displayed in enhanced tracking protection panel to access tracking protection settings -->
+    <string name="etp_settings">Protection Settings</string>
+    <!-- Preference title for enhanced tracking protection settings -->
+    <string name="preference_enhanced_tracking_protection">Enhanced Tracking Protection</string>
+    <!-- Title for the description of enhanced tracking protection -->
+    <string name="preference_enhanced_tracking_protection_explanation_title">Browse without being followed</string>
+    <!-- Description of enhanced tracking protection -->
+    <string name="preference_enhanced_tracking_protection_explanation">Keep your data to yourself. Firefox protects you from many of the most common trackers that follow what you do online.</string>
```

Comment:
Should `Firefox` be the app_name?

Reply:
@mheubusch could we get a confirmation that this string should say "Firefox" and not be translated?